src/index.html

```diff
@@ -13,10 +25,97 @@
     <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/moyo_1.png"
+          alt="Logo"
+        />
+      </a>
+      <nav class="nav-bar">
+        <ul class="nav-list">
+          <li class="item">
+            <a
+              href="index.html"
+              class="is-active nav-link"
```

Comment:
Fix classes everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="index.html"
              class="is-active nav__link"
```